### DISCUSSION TITLE
Correctly highlighting code using Shiki & PHP | Rias.be

### DISCUSSION THREAD

**Discussion by giscus** [riasvdv/rias.be#274] [General]
# Correctly highlighting code using Shiki & PHP | Rias.be

Highlighting code blocks correctly on your website is a more difficult problem to solve than you&#039;d expect. There are many great solutions like Prism or Highlight.js, but they often have difficulty with newer syntax. Inspired by Miguel&#039;s blogpost, I set out to create a PHP wrapper around Sh...

https://rias.be/blog/correctly-highlighting-code-using-shiki-php

**Comment by josueayala27** [comment]
So cool 😜
